src/main/java/frc/robot/autons/AutonSelector.java

```diff
@@ -21,6 +21,7 @@ public enum AutonChoices {
 
     // Choosers
     SendableChooser<AutonChoices> autoChooser;
+    SendableChooser<String> delayChooser;
```

Comment:
```suggestion
    SendableChooser<double> delayChooser;
```
If you do this you don’t need the switch statement. You can just return the get value from the chooser